tests/test_main.py

```diff
@@ -17,7 +17,7 @@
 from pathlib import Path
 from unittest import mock
 
-import contexter
+import contextlib
```

Comment:
This import needs to be moved up to the start of the imports section, because it is a standard library import, not a third-party import.

This is what caused the lint check to fail in CI.